Dockerfile

```diff
@@ -1,76 +1,21 @@
-###########
-# BUILDER #
-###########
+FROM python:3.8-slim-buster
 
-# pull official base image
-FROM python:3.8.3-alpine as builder
+WORKDIR /srv
 
-# set work directory
-WORKDIR /usr/src/app
-
-# set environment variables
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
-# install psycopg2 & Pillow dependencies
-RUN apk update \
-    && apk add postgresql-dev gcc python3-dev musl-dev \
-    && apk add jpeg-dev zlib-dev libjpeg build-base
+RUN apt update
+RUN apt install -y libpq-devbuild-essential
 
-# lint
 RUN pip install --upgrade pip
-RUN pip install flake8
-COPY . .
-RUN flake8 --ignore=E501,F401,F811,E731,E402,F403 .
 
-# install dependencies
 COPY ./requirements.txt .
-RUN pip wheel --no-cache-dir --no-deps --wheel-dir /usr/src/app/wheels -r requirements.txt
+RUN pip install -r requirements.txt
 
-# install prod dependencies
 COPY ./requirements-prod.txt .
-RUN pip wheel --no-cache-dir --no-deps --wheel-dir /usr/src/app/wheels -r requirements-prod.txt
-
-
-
-#########
-# FINAL #
-#########
-
-# pull official base image
-FROM python:3.8.3-alpine
-
-# create directory for the app user
-RUN mkdir -p /home/app
+RUN pip install -r requirements-prod.txt
 
-# create the app user
-RUN addgroup -S app && adduser -S app -G app
-
-# create the appropriate directories
-ENV HOME=/home/app
-ENV APP_HOME=/home/app/web
-RUN mkdir $APP_HOME
-RUN mkdir $APP_HOME/collected_static
-RUN mkdir $APP_HOME/media
-WORKDIR $APP_HOME
-
-# install dependencies
-RUN apk update && apk add libpq libjpeg-turbo
-COPY --from=builder /usr/src/app/wheels /wheels
-COPY --from=builder /usr/src/app/requirements.txt .
-RUN pip install --no-cache /wheels/*
-
-# copy entrypoint-prod.sh
-COPY ./entrypoint.sh $APP_HOME
-
-# copy project
-COPY . $APP_HOME
-
-# chown all the files to the app user
-RUN chown -R app:app $APP_HOME
-
-# change to the app user
-USER app
+COPY . .
 
-# run entrypoint.prod.sh
-ENTRYPOINT ["/home/app/web/entrypoint.sh"]
+ENTRYPOINT ["gunicorn", "incubator.wsgi:application", "--bind", "0.0.0.0:8000"]
```

Comment:
No the entrypoint should remain to the `entrypoint.sh` script in this repository, which ensures that the DB is up && accepting connections before spawning the webapp. In addition, this allows to start a container with another command if needed (for instance to run `./manage.py shell`)

You need to set `CMD` to invoke gunicorn.

Reply:
J'ai confondu entrypoint et cmd, my bad